### ISSUE TITLE
Laracast Theme crashes every time using variables

### ISSUE THREAD

**Issue by krenor** [rainglow/sequel-pro#1]
Steps to reproduce:
* Import [theme](https://github.com/rainglow/sequel-pro/blob/master/themes/Laracasts%20(rainglow).spTheme)
* Type `SET @`
* Typing any further character crashes SequelPro

```
NSInvalidArgumentException

NSConcreteNotifyingMutableAttributedString addAttribute:value:range:: nil value

(
	0   CoreFoundation                      0x00007fffa26582cb __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x00007fffb746e48d objc_exception_throw + 48
	2   CoreFoundation                      0x00007fffa26d6c3d +[NSException raise:format:] + 205
	3   Foundation                          0x00007fffa400baba -[NSConcreteMutableAttributedString addAttribute:value:range:] + 136
	4   UIFoundation                        0x00007fffb528545c __NSConcreteTextStorageLockedForwarding + 49
	5   UIFoundation                        0x00007fffb52857b8 -[NSConcreteTextStorage addAttribute:value:range:] + 84
	6   Sequel Pro                          0x00000001000701a9 -[SPTextView doSyntaxHighlighting] + 1673
	7   Sequel Pro                          0x0000000100072455 -[SPTextView textStorageDidProcessEditing:] + 1136
	8   CoreFoundation                      0x00007fffa25e445c __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 12
	9   CoreFoundation                      0x00007fffa25e435b _CFXRegistrationPost + 427
	10  CoreFoundation                      0x00007fffa25e40c2 ___CFXNotificationPost_block_invoke + 50
	11  CoreFoundation                      0x00007fffa25a1523 -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 1827
	12  CoreFoundation                      0x00007fffa25a055c _CFXNotificationPost + 604
	13  Foundation                          0x00007fffa3fc5677 -[NSNotificationCenter postNotificationName:object:userInfo:] + 66
	14  UIFoundation                        0x00007fffb5283c39 -[NSTextStorage processEditing] + 221
	15  UIFoundation                        0x00007fffb52838e7 -[NSTextStorage endEditing] + 111
	16  AppKit                              0x00007fffa025ddff -[NSTextView insertText:replacementRange:] + 3446
	17  AppKit                              0x00007fffa025ceb7 -[NSTextView insertText:] + 320
	18  Sequel Pro                          0x00000001000a1d18 -[SPNarrowDownCompletion insertAutocompletePlaceholder] + 523
	19  Sequel Pro                          0x0000000100069f08 -[SPTextView doCompletionByUsingSpellChecker:fuzzyMode:autoCompleteMode:] + 5856
	20  Foundation                          0x00007fffa404c4b8 __NSFireDelayedPerform + 417
	21  CoreFoundation                      0x00007fffa25d7c54 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 20
	22  CoreFoundation                      0x00007fffa25d78df __CFRunLoopDoTimer + 1071
	23  CoreFoundation                      0x00007fffa25d743a __CFRunLoopDoTimers + 298
	24  CoreFoundation                      0x00007fffa25ceb81 __CFRunLoopRun + 2065
	25  CoreFoundation                      0x00007fffa25ce114 CFRunLoopRunSpecific + 420
	26  HIToolbox                           0x00007fffa1b2eebc RunCurrentEventLoopInMode + 240
	27  HIToolbox                           0x00007fffa1b2ecf1 ReceiveNextEventCommon + 432
	28  HIToolbox                           0x00007fffa1b2eb26 _BlockUntilNextEventMatchingListInModeWithFilter + 71
	29  AppKit                              0x00007fffa00c5a54 _DPSNextEvent + 1120
	30  AppKit                              0x00007fffa08417ee -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 2796
	31  AppKit                              0x00007fffa00ba3db -[NSApplication run] + 926
	32  AppKit                              0x00007fffa0084e0e NSApplicationMain + 1237
	33  Sequel Pro                          0x0000000100002454 start + 52
	34  ???                                 0x0000000000000001 0x0 + 1
)
```

**Comment by daylerees** [comment]
Woah, that's a weird one! Does it happen with any other themes, or even the build-in ones?

**Comment by daylerees** [comment]
Have managed to replicate this. Will investigate tonight. Thanks!

**Comment by daylerees** [comment]
This is actually quite nasty. I'm using the template given by exporting themes from the app and it still seems to crash. I think I'll consult the sequel pro guys on this one.

**Comment by daylerees** [comment]
Hey buddy, so it was partially me being an idiot with the syntax, and also it's a bug in Sequel Pro for not handling it well. The guys at Sequel Pro are going to take a look at the bug, but I've fixed the syntax in tag `1.5.2` of this repo, so if you use the latest versions it should work fine. Sorry about that, and thanks for bringing it to my attention!
